cfme/roles.py

```diff
@@ -357,8 +361,9 @@ def _remove_from_all(roles, r_page):
         'Compute': {
             'Clouds': ['Instances'],
             'Infrastructure': ['Datastores', 'Providers', 'Virtual Machines', 'Hosts', 'Clusters',
-                               'PXE', 'Resource Pools'],
-            'Physical Infrastructure': ['Providers', 'Racks', 'Switches', 'Servers', 'Topology']},
+                               'Networking', 'PXE', 'Resource Pools'],
+            'Physical Infrastructure': ['Providers', 'Chassis', 'Racks', 'Switches', 'Servers',
+                                        'Storages', 'Topology']},
         'Control': ['Explorer', 'Log', 'Simulation'],
```

Comment:
Q: Why no `Import/Export` for `Control` here?

Reply:
Because there is no 'Import/Export' option under 'Control' when you are logged in as a member of group 'evmgroup-auditor'.
You can see for yourself by creating new user in CFME and assigning group 'Evmgroup-auditor' to the user, then logging in as newly created user and navigating to 'Control'.

Reply:
Makes sense -- thanks for the response!